### ISSUE TITLE
Uncaught SyntaxError: Unexpected token <

### ISSUE THREAD

**Issue by rmbl** [ff00ff/koala-atom-package#1]
While editing css and slim files

**Atom Version**: 0.165.0
**System**: Mac OS X 10.10.1
**Thrown From**: [koala](https://github.com/ff00ff/koala-atom-package) package, v0.4.1

### Stack Trace

Uncaught SyntaxError: Unexpected token <

```
At file:///opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app/static/index.html?loadSettings=%7B%22bootstrapScript%22%3A%22%2Fopt%2Fhomebrew-cask%2FCaskroom%2Fatom%2Flatest%2FAtom.app%2FContents%2FResources%2Fapp%2Fsrc%2Fwindow-bootstrap.js%22%2C%22resourcePath%22%3A%22%2Fopt%2Fhomebrew-cask%2FCaskroom%2Fatom%2Flatest%2FAtom.app%2FContents%2FResources%2Fapp%22%2C%22devMode%22%3Afalse%2C%22safeMode%22%3Afalse%2C%22appVersion%22%3A%220.165.0%22%7D:1

SyntaxError: Unexpected token <
  at Object.parse (native)
  at IncomingMessage.<anonymous> (/Users/rmbl/.atom/packages/koala/lib/packet.js:44:20)
  at IncomingMessage.emit (events.js:129:20)
  at _stream_readable.js:907:16
  at process._tickCallback (node.js:378:11)

```

### Commands

```
  2x -0:07.8 core:select-down (input.hidden-input)
     -0:07.0 editor:select-to-end-of-line (input.hidden-input)
     -0:06.5 core:copy (input.hidden-input)
  2x -0:06.2 core:move-up (input.hidden-input)
     -0:05.7 core:move-down (input.hidden-input)
     -0:05.4 editor:newline (input.hidden-input)
     -0:05.1 core:move-up (input.hidden-input)
     -0:04.7 editor:newline (input.hidden-input)
     -0:04.5 core:paste (input.hidden-input)
  2x -0:03.8 core:move-up (input.hidden-input)
     -0:03.1 core:move-down (input.hidden-input)
     -0:02.9 core:move-right (input.hidden-input)
     -0:02.6 editor:move-to-end-of-screen-line (input.hidden-input)
     -0:02.3 core:move-left (input.hidden-input)
  5x -0:01.7 core:backspace (input.hidden-input)
     -0:00.0 core:save (input.hidden-input)
```

### Config

```
{
  "core": {
    "themes": [
      "seti-ui",
      "seti-syntax"
    ],
    "disabledPackages": [
      "vim-mode"
    ]
  },
  "koala": {
    "email": "philipp@gildein.com",
    "accessToken": "c82ab2227c41b910872a5c0793309eaaa8e7f4e3d3de4c021acd6ff1766c8018caae77c81094a37b55448f3ec28ec60a07eda356284e20298473353469a028633198c2180ca8df9f2707791ab135a99142ee6675f93ac169426a9b1a69ea919e53a775bc934da665d4c8840f457f5f913cd630d7412bd30a99f4faae2ac31592"
  }
}
```

### Installed Packages

```coffee
# User
editorconfig, v0.2.2
highlight-line, v0.9.2
koala, v0.4.1
language-dart, v0.1.1
language-latex, v0.4.1
language-rspec, v0.2.1
language-slim, v0.3.0
linter, v0.9.0
linter-coffeelint, v0.1.7
linter-csslint, v0.0.11
linter-erb, v0.0.3
linter-htmlhint, v0.0.8
linter-jshint, v0.1.0
linter-ruby, v0.1.4
seti-syntax, v0.2.1
seti-ui, v0.5.1

# Dev
No dev packages
```

/cc @atom/core

**Comment by lindseymysse** [comment]
This issue is now affecting me. Atom now issues 2 new error message s when ever I switch panels. This makes Atom unusable. 

Bump.

**Comment by martijndeh** [comment]
I'm looking into this right now.

**Comment by martijndeh** [comment]
Every now and then when getkoala.co went offline, the Atom plugin would reject a promise and Atom would log this error. I published an update to the Atom package, 0.5.0, which catches this error which prevents it from bubbling to Atom. This means no more annoying popups.

There is no data loss when getkoala.co is offline as the events are stored locally incase syncing fails.
